src/reducers/neighbourhoodSlice.ts

```diff
@@ -9,10 +9,6 @@ interface updateZone {
   newZoning: string;
 }
 
-interface MappedNeighbourhood {
-  [key: number]: INeighbourhood;
-}
-
```

Comment:
This does not break O(1) search. Instead of being an object where its 

{0: N1, 1:  N2}

its just [N1, N2].

I gave a quick try on both master and my branch and it doesnt feel different

Reply:
I added changes to reflect this